canteens/studierendenwerk.py

```diff
@@ -115,14 +116,12 @@ def parse_menu(menu_html):
             price = item.find('div', class_='text-right').text.strip()
             price_exp = re.compile(r'€ (\d,\d+).*$')
             price = price_exp.sub('*\g<1>€*', price)
-            text = '%s%s %s: %s\n' % (text, annotation, title, price)
-    if text.strip() == '':
-        return ''
-    else:
-        lines = text.split('\n')
-        lines.sort()
-        text = '\n'.join(lines)
-        return '*Speiseplan*%s' % text
+            group_text += '%s %s: %s\n' % (annotation, title, price)
+
+        if len(group_text.strip()) > 0:
+            group_heading = group.find("div").find("div").text.strip()
+            text += f"*{group_heading}*\n{group_text}"
```

Comment:
I understand your intention to use the f-strings as a newer implementation of formatted strings. But I do not like to use different implementations in the same code. Let's make it consistent with the rest of the code.

```suggestion
            text += '\n*%s*\n%s' % (group_heading, group_text)
```

I also added an additional newline before the `group_heading` in my suggestion. I think it makes the output a lot better readable. Especially for Telegram clients without support for **bold texts** - like the official linux desktop client :man_shrugging:.

Reply:
If you implement my idea from line 119 you need to add two newlines before every `group_heading`. :thinking: